pkg/kube/controllers/extendedstatefulset/reconciler.go

```diff
@@ -31,6 +31,7 @@ import (
 	"code.cloudfoundry.org/cf-operator/pkg/kube/util/finalizer"
 	"code.cloudfoundry.org/cf-operator/pkg/kube/util/owner"
 	"code.cloudfoundry.org/cf-operator/pkg/kube/util/versionedsecretstore"
+	"code.cloudfoundry.org/cf-operator/testing"
```

Comment:
@edwardstudy , kubectlHelper is mostly used for testing in the e2e tests. So, I put it in testing folder instead of utils. KubectlHelper can be thought of just like the machine file.

Reply:
@rohitsakala we shouldn't reference test code - test code is likely not tested